content/en/seachat/seachat-manual/02-create-agent/03-agent-duplicaiton.md

```diff
@@ -45,3 +45,36 @@ This process might take some time based on the complexity of the agent. Once you
 <br/>
 
 For users with integrations or channels connected to the original agent, you will need to update the integration to include the new agent copy since the agent copy is still a new agent.
+
+## Agent Replacement
+
+The agent replacement feature provides an efficient way to manage different versions of your agent.
+
+This is particularly useful when you want to develop an existing agent without affecting the original version. By using this feature, agent editors can make changes to a copy of the original agent rather than modifying the original agent directly.
+
+<br/>
+<center>
+<a style="border-radius: 0.4rem; cursor: zoom-in;" href="/images/seachat/en/create-a-new-agent/replace-button.png" target="_blank">
+<img width="80%" style="border-radius: 0.4rem" src="/images/seachat/en/create-a-new-agent/replace-button.png" alt="">
+</a>
+
+**Agent Replacement**
+</center>
+
+A common use case might look like this:
+
+1. Duplicate the original Agent A and name the copy Agent B (Agent A Copy).
+2. Add new features to Agent B and test them.
+3. Once development is complete, replace Agent A with Agent B.
+4. Now, the new agent is running in production with the original settings from Agent A and the new features implemented in Agent B.
+
+<br/>
+<center>
+<a style="border-radius: 0.4rem; cursor: zoom-in;" href="/images/seachat/en/create-a-new-agent/replacement-workflow.jpg" target="_blank">
+<img width="80%" style="border-radius: 0.4rem" src="/images/seachat/en/create-a-new-agent/replacement-workflow.jpg" alt="">
+</a>
+
+**Agent Replacement Workflow**
+</center>
+
+By following this process, you can keep the original agent running in production while developing and testing a new version of the agent.
```

Comment:
add a small section called "What's replaced?" -- basic info, advanced settings, knowledge base, widget, but not channels